libs/SalesforceReact/build.gradle.kts

```diff
@@ -23,8 +23,8 @@ plugins {
 
 dependencies {
     api(project(":libs:MobileSync"))
-    api("com.facebook.react:react-native:0.70.14")
-    implementation("androidx.core:core-ktx:1.12.0")
+    api("com.facebook.react:react-android:0.72.6")
```

Comment:
SalesforceMobileSDK-ReactNative is on 0.72.7

Reply:
I'm about to commit an uplift to 0.72.7 while I'm at it. 👍🏻